src/main/resources/mycore-classifications/coar_resource_types_31.xml

```diff
@@ -0,0 +1,380 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mycoreclass xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:noNamespaceSchemaLocation="MCRClassification.xsd" ID="coarResourceTypes">
+  <label xml:lang="de" text="COAR Resource Types 3.1"/>
```

Comment:
Gibt es hier auch ein deutsches Label für die Klassifikation?

Reply:
Konnte ich nicht finden, in der Spezifikation steht nichts: https://vocabularies.coar-repositories.org/resource_types/